ubuntu22.04/Dockerfile

```diff
@@ -80,16 +80,22 @@ RUN if [ "$DRIVER_TYPE" != "vgpu" ]; then \
     cd drivers && \
     /tmp/install.sh download_installer; fi
 
-# Fabric manager packages are not available for arm64
-RUN if [ "$DRIVER_TYPE" != "vgpu" ] && [ "$TARGETARCH" != "arm64" ]; then \
+RUN if [ "$DRIVER_TYPE" != "vgpu" ]; then \
     apt-get update && \
     apt-get install -y --no-install-recommends nvidia-fabricmanager-${DRIVER_BRANCH}=${DRIVER_VERSION}-1 \
     libnvidia-nscq-${DRIVER_BRANCH}=${DRIVER_VERSION}-1 && \
     rm -rf /var/lib/apt/lists/*; fi
 
 RUN if [ "$DRIVER_TYPE" != "vgpu" ] && [ "$DRIVER_BRANCH" -ge "550" ]; then \
     apt-get update && \
-    apt-get install -y --no-install-recommends nvidia-imex-${DRIVER_BRANCH}=${DRIVER_VERSION}-1 && \
+    apt-get install -y --no-install-recommends nvlsm \
```

Comment:
Question: is there a specific version of nvlsm we need to install? Is it tightly coupled to the driver version? Or is it desired to pick up the latest?

Reply:
nvlsm doesn't seem to be pinned to any particular when you look it up [here](https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64/)